metrics-datadog/src/test/java/org/coursera/metrics/datadog/DatadogReporterTest.java

```diff
@@ -153,6 +153,29 @@ public void reportsDoubleGaugeValues() throws Exception {
   }
 
   @Test
+  public void reportsGaugeValues_ThrowsException_OtherMetricsWillBeReported() throws Exception {
```

Comment:
Though the styleguide permits underscores in tests, the project isn't using them. Perhaps `reportHandlesGaugeMetricExceptions`?